src/main/java/frc/robot/ControllerBindings.java

```diff
@@ -9,18 +9,23 @@
 
     private LogitechDualAction driverPad;
     private LogitechDualAction operatorPad;
+    private int driverPort;
+    private int operatorPort;
 
     public static final LogitechControl DRIVE_LEFT_STICK = LogitechControl.LEFT_STICK;
     public static final LogitechControl DRIVE_RIGHT_STICK = LogitechControl.RIGHT_STICK;
 
-    ControllerBindings(int driverPort, int operatorPort){
+    ControllerBindings(){
         // Driver Pad
-        this.driverPad = new LogitechDualAction(driverPort);
+        this.driverPort = 0;
```

Comment:
Should probably be a `static final int NAMED_CONSTANT` like the sticks currently are, so that they can be easily found at the top of the file. Although, these are probably the least likely to change out of all of these constants.